tests/unit/named.bounds.test.scala

```diff
@@ -0,0 +1,28 @@
+// SYNTAX TEST "source.scala"
+
+   def fromInt[T: Numeric as num](t: Int): T = num.fromInt(t)
```

Comment:
Could you add a test with the new curly braces syntax for context bounds?

Reply:
Done!